third-party/dropbear/stubs.h

```diff
@@ -2,17 +2,22 @@
  * @file
  * @brief
  *
- * @author  Anton Kozlov
- * @date    17.04.2014
+ * @author  Anton Kozlov & Mark Kovalev
+ * @date    21.11.2018
  */
 
-#ifndef DROPBEAR_2014_63_STUBS_H_
-#define DROPBEAR_2014_63_STUBS_H_
+#ifndef DROPBEAR_STUBS_H_
+#define DROPBEAR_STUBS_H_
 
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
 
+#define DROPBEAR_URANDOM_DEV "/etc/dropbear/random"
+#define _PATH_TTY "/dev_pty_0"
```

Comment:
This macro is defined here and in `dropbear-2018.76/sysoptions.h`. Is it really necessary?